File: src/test/java/com/tartner/utilities/SequentialGuidGeneratorTest.java
package com.tartner.utilities;

import org.junit.Test;

import java.util.UUID;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.notNullValue;

public class SequentialGuidGeneratorTest {
    @Test
    public void newId() throws Exception {
        SequentialGuidGenerator generator = new SequentialGuidGenerator();

        for( int i = 0; i < 100; i++ ) {
            final UUID id = generator.newId();
//            System.out.println( id );
            assertThat( id, notNullValue() );
        }
    }

}

File: src/test/java/com/tartner/utilities/SequentialGuidGeneratorTest.kt
package com.tartner.utilities

import org.hamcrest.MatcherAssert.assertThat
import org.hamcrest.Matchers.notNullValue
import org.junit.Test

public class SequentialGuidGeneratorTest {
    @Test @throws(Exception::class)
    public fun newId() {
        val generator = SequentialGuidGenerator()

        for (i in 0..99) {
            val id = generator.newId()
            //            System.out.println( id );
            assertThat(id, notNullValue())
        }
    }

}
